manifests/local-storage.yaml

```diff
@@ -124,7 +124,8 @@ data:
             ;;
         esac
     done
-    mkdir -m 0700 -p ${absolutePath}
+    mkdir -m 0777 -p ${absolutePath}
+    chmod 701 $(dirname ${absolutePath})
```

Comment:
Will a user be be able to list contents with just exec on the directory?

Reply:
what about
```suggestion
    chmod 701 ${absolutePath}/..
```

Reply:
I don't expect this to be an issue in practice, but is it possible to set `default-local-storage-path` to `/`? If so, strange things may happen.

In my opinion, such a weak security mechanism is not worth the hassle.

Reply:
The local path provisioner isn't intended to be a real enterprise-class CSI provider; it's "Just Enough" to get PVCs working in a standalone/dev environment.

And yes, there are probably many things that you could set to a value that would cause bad thing to happen to your system. Setting the local storage to / is one of them.

Reply:
@briandowns The user cannot list the contents of the directory. I tried with the following:
- `ls`
- `find . ! -name . -prune`
- `echo *`
- `printf '%s\n' .* *`
- `lsattr`

All failed to give me any files.

Reply:
Fixed suggestion and added an integration test.